eng/scripts/invoke_embedme.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Python version 3.4 or higher is required to run this script.
+
+# Use case: Invokes embedme README codesnippet generation and validation regardless of OS.
+#
+# Flags
+#   --readme/-r: Path to the README.
+#   --verify/-v: Flag indicating to only perform a dry-run validation.
+#   --debug/-d: Flag indicating to perform debug level logging.
+#
+# For example: Generating README codesnippets for Azure Storage Blobs.
+#    python eng/scripts/invoke_embedme.py -r sdk/storage/azure-storage-blob/README.md
+#
+# For example: Valdate README codesnippets for Azure Core.
+#    python eng/scripts/invoke_embedme.py -r sdk/core/azure-core/README.md -v
+#
+# The script must be run at the root of azure-sdk-for-java.
+
+import argparse
+import os
+import sys
+
+# NPX command for Windows OS.
+windows_command = 'npx.cmd'
```

Comment:
On windows you can usually execute a ".cmd" file by simply calling the name of the file such as "npx" so it might be worth just executing npx in both OS contexts.

Reply:
Simplified!